chain/bitcoind_conn.go

```diff
@@ -174,6 +175,14 @@ func NewBitcoindConn(cfg *BitcoindConfig) (*BitcoindConn, error) {
 		}
 	}
 
+	// Check if the bitcoind node is on a version that has the
+	// gettxspendingprevout RPC. If it does, then we don't need to maintain
+	// a mempool for ZMQ clients.
+	hasRPC, err := hasSpendingPrevoutRPC(client)
```

Comment:
What about the non-zmq variant? In that case, the mempool is only useful for detecting unconf transactions, and we don't need to periodically reconcile.

Reply:
It already has to poll the mempool and build a local one to not give duplicate notifications, so the call to `LookupInputSpend` just checks the btcwallet mempool. For that reason, I don't think it makes sense to use the new RPC?